packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -52,119 +62,251 @@ class _DeepLinkPageState extends State<DeepLinkPage>
 
   @override
   Widget build(BuildContext context) {
-    return ListView(
-      children: [
-        Row(
-          children: [
-            Expanded(
-              child: Text(
-                'All deep links',
-                style: Theme.of(context).textTheme.bodyLarge,
+    return DefaultTabController(
+      length: 3,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            children: [
+              Expanded(
+                child: Text(
+                  'All deep links',
+                  style: Theme.of(context).textTheme.bodyLarge,
+                ),
               ),
-            ),
-            InkWell(
-              onTap: () {
-                controller.bundleByDomain = !controller.bundleByDomain;
-              },
-              child: const Text('Bundle by domain'),
-            ),
-            const SizedBox(width: denseSpacing),
-            SearchBar(
-              leading: const Icon(Icons.search),
-              onChanged: (value) {
-                controller.searchContent = value;
-              },
-              constraints: BoxConstraints.tight(
-                Size(defaultSearchFieldWidth, defaultTextFieldHeight),
+              const SizedBox(width: denseSpacing),
+              SearchBar(
+                leading: const Icon(Icons.search),
+                hintText: 'Search a URL, domain or path',
+                onChanged: (value) {
+                  controller.searchContent = value;
+                },
+                constraints: BoxConstraints.tight(
+                  Size(wideSearchFieldWidth, defaultTextFieldHeight),
+                ),
+              ),
+            ],
+          ),
+          const SizedBox(height: denseSpacing),
+          const TabBar(
```

Comment:
consider using `AnalyticsTabbedView` so that you will automatically get analytics reported for tab switches. This would be good information to know which tab users are using the most.

Reply:
this comment still stands as well

Reply:
I think I will add analytics in a separated PR